src/App.vue

```diff
@@ -490,14 +521,24 @@ export default {
 
             this.sections = sections;
 
-            this.closeImportExport();
+            this.closeImport();
+        },
+
+        closeImport () {
+            this.importModal.data = '';
+            this.$modal.hide('importModal');
+        },
+
+        closeExport () {
+            this.exportModal.data = '';
+            this.$modal.hide('exportModal');
         },
 
-        closeImportExport () {
-            this.isImport = false;
-            this.importExportModal.data = '';
+        copyToClipboard () {
+            document.getElementById("export").select();
```

Comment:
Instead of manipulating the DOM with `document.getElementById`, use v-ref: https://vuejs.org/v2/guide/components.html#Child-Component-Refs